README.md

```diff
@@ -449,6 +449,7 @@ jobs:
         imagepullsecrets: |
           demo-k8s-secret
 ```
+k8s-deploy GitHub Actions is supported on both 'AzureUSGovernment' or 'AzureChinaCloud'.Login to the respective Azure Cloud before running this action using [Azure Login](https://github.com/Azure/login)

```

Comment:
K8s-deploy doesn't depend on `az login`

Reply:
The statement to use az login is removed